agent/config/types.go

```diff
@@ -83,6 +83,10 @@ type Config struct {
 	// other than containers managed by ECS
 	ReservedMemory uint16
 
+	// ContainerTimeout specifies the amount time before a SIGKILL is issued to
+	// containers managed by ECS
+	DockerStopTimeoutSeconds uint64
```

Comment:
I think my preference would be to have the type be a `time.Duration` and use `time.ParseDuration` for parsing.